ambassador/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: ambassador-edge-stack
+title: "Ambassador Edge Stack"
+version: "latest"
```

Comment:
We are moving away from using the latest version, can we put a stable version for this ?

Reply:
Done!